base/applications/rapps/gui.cpp

```diff
@@ -1850,6 +1850,29 @@ class CMainWindow :
 
         return CWindowImpl::Create(NULL, r, szWindowName.GetString(), WS_OVERLAPPEDWINDOW | WS_CLIPCHILDREN | WS_CLIPSIBLINGS, WS_EX_WINDOWEDGE);
     }
+
+    void HandleTabOrder(int direction)
+    {
+        HWND Controls[] = { m_Toolbar->m_hWnd, m_SearchBar->m_hWnd, m_TreeView->m_hWnd, m_ListView->m_hWnd, m_RichEdit->m_hWnd };
```

Comment:
This looks hardcoded; isn't it possible to find this list by listing the child windows of the main window?

Reply:
It looks hardcoded because it is.
Sure we could make it more complicated, but then we'd need to take into account all kinds of bullshit rules:
Don't include the splitters, the search box is inside the toolbar and whatnot.